### ISSUE TITLE
Syntax highlighting contrast

### ISSUE THREAD

**Issue by mal-tee** [datenanfragen/website#491]
According to Firefox the red chars don't have enought contrast, e.g. on https://www.datarequests.org/blog/honey-data-collection/

![grafik](https://user-images.githubusercontent.com/45886910/98441367-0b66ca80-20fe-11eb-9a59-0c9ab40aa748.png)

**Comment by Bhavin789** [comment]
Hi @mal-tee @baltpeter , if you can give me a few more details on what is expected, would love to give it a try. Thanks!

**Comment by baltpeter** [comment]
@Bhavin789 What exactly are you unclear on?

**Comment by Bhavin789** [comment]
@baltpeter I meant from where I can see the above Screenshot? Apology if I am missing something. I am able to run the website locally but don't know how to check for the above screenshot.

Do we show code snippets anywhere? Thanks
